internal/hooks/v0hooks/v0http/v0http.go

```diff
@@ -0,0 +1,276 @@
+package v0http
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"mime"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/gofrs/uuid"
+	"github.com/sirupsen/logrus"
+	"github.com/supabase/auth/internal/api/apierrors"
+	"github.com/supabase/auth/internal/conf"
+	"github.com/supabase/auth/internal/observability"
+
+	standardwebhooks "github.com/standard-webhooks/standard-webhooks/libraries/go"
+)
+
+const (
+	defaultHTTPHookTimeout  = 5 * time.Second
```

Comment:
seems like the [previous timeout duration was 2 seconds](https://github.com/supabase/auth/pull/1994/files#diff-070d7377dd35255e1da96c9252a425b4f731c54d85f3c7675bd3a3cff522e5c3L16-L20) and we're increasing it to 5 seconds here? 

shouldn't be an issue but just want to call out the change

Reply:
There was actually two constants, `defaultTimeout` which was used in pgfunc and then this timeout used in HTTP. Each has been preserved in the new packages.